### DISCUSSION TITLE
Safe exam browser does not read/load the whole page, why?

### DISCUSSION THREAD

**Discussion by isotopica** [SafeExamBrowser/seb-win-refactoring#879] [Q&A]
Like 10 out of 20 students does not get the whole site loaded in SEB. Opening at the same time. It's not network related, what could it be?
It works if the same students open the page in Microsoft Edge. Could it be that SEB is sensitive to the fact that if it takes "too long" to load a part, that it doesn't load it at all? Its not a heavy site.
 I'm not sure but the module not loading i think is in java(script)?

**Comment by dbuechel** [comment]
If there are any issues while loading a page, then they should be logged in the client and/or browser logs, to be found under `%LocalAppData%\SafeExamBrowser\Logs`.
